# type can be simple, weighted, symmetric or filter
# Config supports one default setting for all files with different options for first and last in sequence
# To get fully custom settings for all files, the python script must be modified and the script called multiple times
#   with different yaml configurations
moving_average:
  default:
    type: "simple"
    window_size: [2, 2, 2, 2, 4, 4, 1]
    keep_first: false
    keep_last: false
  first:
    type: "symmetric"
    window_size: [2, 2, 2, 2, 4, 4, 1]
    keep_first: false
    keep_last: true
  last:
    type: "simple"
    window_size: [2, 2, 2, 2, 4, 4, 1]
    keep_first: false
    keep_last: true
    
